### PR TITLE
Fix ICA tests between versions v0.3.2 and v0.1.3

### PR DESCRIPTION
**Pull request by chatton**
## Description

There was an issue where the v0.1.3 -> v0.3.2 was failing because the controller chain was on a higher version than the host chain, in this case if the version string is left empty it will default to an incentivized channel creation when we register an interchain account which will not be supported yet on the host chain.

Note: the failures in the below tests are expected as incentivized channels are not supported/

#### TestInterchainAccountsTestSuite
  - [x] [chain A (v0.3.2) -> chain B (v0.1.3)](https://github.com/cosmos/ibc-go/actions/runs/2910127474)
  - [x] [chain A (v0.1.3) -> chain B (v0.3.2)](https://github.com/cosmos/ibc-go/actions/runs/2910127816)
  - [x] [chain A (v0.2.2) -> chain B (v0.3.2)](https://github.com/cosmos/ibc-go/actions/runs/2910128183)
  - [x] [chain A (v0.3.2) -> chain B (v0.2.2)](https://github.com/cosmos/ibc-go/actions/runs/2910128559)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by charleenfei — APPROVED** [review]

File: e2e/testconfig/testconfig.go [hunk 1]
```diff
@@ -90,6 +90,21 @@ func FromEnv() TestConfig {
...
+	}
+	return chainATag
+}
+func GetChainBTag() string {
+	chainBTag, ok := os.LookupEnv(ChainBTagEnv)
+	if !ok {
+		return GetChainATag()
+	}
+	return chainBTag
+}
```
Comment (damiannolan) [inline e2e/testconfig/testconfig.go:106]: mega nit - spacing

```suggestion
func GetChainATag() string {
	chainATag, ok := os.LookupEnv(ChainATagEnv)
	if !ok {
		panic(fmt.Sprintf("no environment variable specified for %s", ChainATagEnv))
	}
	return chainATag
}

func GetChainBTag() string {
	chainBTag, ok := os.LookupEnv(ChainBTagEnv)
	if !ok {
		return GetChainATag()
	}
	return chainBTag
}
```

File: e2e/testsuite/testsuite.go [hunk 2]
```diff
@@ -31,6 +31,8 @@ const (
 	ChainARelayerName = "rlyA"
 	// ChainBRelayerName is the name given to the relayer wallet on ChainB
 	ChainBRelayerName = "rlyB"
+
+	emptyLogs = "[]"
```
Comment (damiannolan) [inline e2e/testsuite/testsuite.go:35]: should we add a super basic godoc for this?

**Review by damiannolan — APPROVED** [review]
Nice!

**Review by seantking — APPROVED** [review]

**Review by colin-axner — APPROVED** [review]

File: e2e/testsuite/testsuite.go (hunk 2 above)
Comment (chatton) [reply e2e/testsuite/testsuite.go:35]: yep good idea I'll add one

**Pull request merged**